src/Service/LegacyModListImport/ModListImportHtml.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service\LegacyModListImport;
+
+use App\Service\LegacyModListImport\Dto\ModInfo;
+use App\Service\LegacyModListImport\Dto\ModlistInfo;
+use Symfony\Component\DomCrawler\Crawler;
+
+class ModListImportHtml
+{
+    public function importFromDirectoryHtml(string $path): ModlistInfo
```

Comment:
Introduce a common `ModListReaderInterface`:
```php
interface ModListReaderInterface
{
    public function read(string $source): ModList
}
```
and then provide two implementations for this interface: `ModListCsvReader` and `ModListHtmlReader`. In both cases our `$source` will be path to the file. `ModListCsvReader` can be based on `ModListImport` and `ModListCsvReader` services. This will require a little bit of refactoring of both existing and new classes.